assets/js/components/GoogleChart/index.js

```diff
@@ -64,6 +69,7 @@ export default function GoogleChart( props ) {
 		children,
 		className,
 		data,
+		dateMarkers,
```

Comment:
I wonder if it's worth filtering the `dateMarkers` so we don't try to render them at all of their date is outside of the current date range... I think that would be preferable to taking the approach that we do now where the marker is attempted to be rendered somewhere out of the visible range.

Reply:
That _did_ totally fix the VRT issues, thanks! 🙏🏻